[2020-02-06 11:04:22,074] test_config.py->create_default_params line:21 [ERROR]Creating default configuration
[2020-02-06 11:04:22,075] test_config.py->read_config_file line:42 [ERROR]read test config from file C:\PAT\LEN_AUTO_19.00_DEV01_MODEM_V1.0\02_config\test_config.ini
[2020-02-06 11:04:22,075] test_config.py->read_config_file line:56 [INFO]{'project': 'Defender', 'device_name': 'KeysightUXM', 'tc_module': 'Modem_CA_Verify_Keysight', 'test_config_name': 'test_config.ini', 'ProjectName': 'Racer', 'is_restart_phone': 'True', 'root_path': 'C:\\PAT\\LEN_AUTO_19.00_DEV01_MODEM_V1.0', 'config_path': 'C:\\PAT\\LEN_AUTO_19.00_DEV01_MODEM_V1.0\\02_config', 'CA_tpye': 'LTE', 'ca_combs_name': 'ca_combs.xlsx', 'ca_parameter_load_type': 'manually', 'document_path': 'C:\\PAT\\LEN_AUTO_19.00_DEV01_MODEM_V1.0\\03_document', 'auto_get_qualcomm_port': 'True', 'qualcomm_port': '21', 'phone_control_method': 'adb', 'is_loop_pcell': 'False', 'device_ip': 'TCPIP0::192.168.0.2::hislip2::INSTR', 'device2_name': 'CMW500', 'GPIB': 'GPIB0::20::INSTR'}
[2020-02-06 11:04:22,842] main.py->get_tc_module line:33 [INFO]<module 'modules.Modem_CA_Verify_Keysight' from 'C:\\PAT\\LEN_AUTO_19.00_DEV01_MODEM_V1.0\\modules\\Modem_CA_Verify_Keysight.py'>
[2020-02-06 11:04:22,842] Modem_CA_Verify_Keysight.py->__init__ line:32 [INFO]*** module Modem_CA_Verify_Keysight constructor
[2020-02-06 11:04:22,843] Modem_CA_Verify_Keysight.py->open_test_report_csv_file line:146 [INFO]open CSV File:C:\PAT\LEN_AUTO_19.00_DEV01_MODEM_V1.0\01_logs_2020-02-06_110422\CA_TEST.csv
[2020-02-06 11:04:22,844] Modem_CA_Verify_Keysight.py->main line:189 [INFO]########################################
[2020-02-06 11:04:22,844] Modem_CA_Verify_Keysight.py->main line:190 [INFO]#########main starts#########
[2020-02-06 11:04:22,844] Modem_CA_Verify_Keysight.py->main line:191 [INFO]########################################
